README.md

```diff
@@ -4,16 +4,64 @@ Model Assessment and Decision Analysis for Marine Environments web application.
 Prototype Decision Support app for the [ADRIA.jl](https://github.com/open-AIMS/ADRIA.jl)
 platform and CoralBlox model. This project also contains the Reef Guide app.
 
+## Table of Contents
+- [Quick Start](#quick-start)
+- [Angular Developer Setup](#angular-developer-setup)
+- [Development Server](#development-server)
+- [ReefGuide Web API Integration](#reefguide-web-api-integration)
+- [Code Scaffolding](#code-scaffolding)
+- [Configuration](#configuration)
+- [Build](#build)
+- [Deploy](#deploy)
+- [Testing](#testing)
+- [Further Help](#further-help)
+
+## Quick Start
+
+```bash
+# Clone the repository
+git clone https://github.com/open-AIMS/MADAME-app.git
+cd MADAME-app
+
+# For Linux: Install Node with NVM
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
+source ~/.bashrc
+nvm install 22
+nvm use 22
+
+# Install dependencies
+npm install --legacy-peer-deps
+
+# Start development server
+ng serve
+
+# Access the application at http://localhost:4200/
+```
+
 ## Angular Developer Setup
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli)
 version 18.0.6.
 
 **One-time setup:**
 1. Install [Node v22](https://nodejs.org/en/download/package-manager)\
```

Comment:
This URL seems to have changed. It redirects to https://nodejs.org/en/download now